README.md

```diff
@@ -0,0 +1,70 @@
+# operator-framework-tooling
+
+This repository contains tools for downstreaming OLM software
+
+## OLMv0
+
+The OLMv0 code is downstreamed into the [openshift/operator-framework-olm](https://github.com/openshift/operator-framework-olm) repository from the following upstream repositories:
+
+* [operator-framework/api](https://github.com/operator-framework/api)
+* [operator-framework/operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager) - the main repo
+* [operator-framework/operator-registry](https://github.com/operator-framework/operator-registry)
+
+The `operator-lifecycle-manager` repository is the main repository, as it refers to the others. The its `go.mod` file determines what versions of `api` and `operator-registry` are used and downstreamed. Any integration between these three repositories _must_ be done upstream, as that will provide a clean downstreaming experience.
+
+The upstream repositories must also be `go mod tidy` clean.
+
+The downstream repository is a "monorepo" consisting of the above reposities located in a `staging` directory. The upstream commits are cherry-picked with updating comments indicating the source repository and commit SHA.
+
+## OLMv1
+
+The OLMv1 code is downstreamed into the [openshift/operator-framework-operator-controller](https://github.com/openshift/operator-framework-operator-controller) repository from the following upstream repository:
+
+* [operator-framework/operator-controller](https://github.com/operator-framework/operator-controller)
+
+There is only one repository that is downtreamed. The upstream commits are downstreamed as a direct mirror (i.e. commit SHAs remain the same) and then merged into the `main` branch.
```

Comment:
I don't think this is true. There are downstream equivalents of catalogd + rukpak that are currently downstreamed by the syncer.

I believe the process is:
- Check for updates to catalogd + rukpak based on operator-controller go.mod
- If updates for catalogd + rukpak, sync them, don't sync operator-controller yet
- If no updates for catalogd + rukpak, sync operator-controller

Reply:
I haven't done any syncing for v1... but my understanding is that it's just `operator-controller` code that's merged, and catalogd + rukpak are vendored?

Reply:
I'm looking at the code, and it seems to merge all three, but only update go.mod of operator-controller if the others don't need changes. I'm not seeing how it's _not_ syncing operator-controller...

Reply:
That part that I'm saying I don't think is true is that it _only_ syncs operator-controller

Reply:
Sync PR of catalogd: https://github.com/openshift/operator-framework-catalogd/pull/46

Reply:
Right; I've updated the PR.